include/video.h

```diff
@@ -72,7 +72,12 @@ void GFX_SwitchFullScreen(void);
 bool GFX_StartUpdate(uint8_t * &pixels, int &pitch);
 void GFX_EndUpdate( const Bit16u *changedLines );
 void GFX_GetSize(int &width, int &height, bool &fullscreen);
-void GFX_LosingFocus(void);
+void GFX_LosingFocus();
+void GFX_RegenerateWindow(Section *sec);
+
+bool wants_stretched_pixels();
+void remove_window();
+void set_output(Section *sec, bool should_stretch_pixels);
```

Comment:
Thanks @Wengier. 
Hopefully these three prototypes (wants_stretched_pixels, remove_window, set_output) can now be dropped from the `video.h` header?

Reply:
Removed them from the header. Done.